### DISCUSSION TITLE
Toggle for showing inaccuracies in analysis for just you VS you+opponent

### DISCUSSION THREAD

**Discussion by ijm8710** [lichess-org/mobile#816] [Ideas]
Generally I prefer to first glean my own errors. Then sometimes I like to see errors we both made.

As is, having them both auto-included makes it hard to sift through quickly.

Is this possible and do we see value to this?

**Comment by EmmetSchuler** [comment]
I think this function will come with the learn from your mistakes feature.

**Reply by ijm8710** [reply]
Think you're right. Thanks!
